src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,27 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+
+class Centro(ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun addVendedor(vendedor: Vendedor) {
+        if (this.vendedores.contains(vendedor)) {
+            throw Exception("Este vendedor ya trabaja en el centro de distribucion")
+        }
+        else {
+            this.vendedores.add(vendedor)
+        }
+    }
+
+    fun vendedorEstrella() = this.vendedores.maxBy { it.puntajeCertificaciones() }
+
+    fun puedeCubrir(ciudad: Ciudad) = this.vendedores.any { it.puedeTrabajarEn(ciudad) }
+
+    fun vendedoresGenericos() = this.vendedores.filter { it.esGenerico() }
+
+    fun esRobusto() = this.vendedoresFirmes().size >= 3
+
+    fun vendedoresFirmes() = this.vendedores.filter { it.esFirme() }
```

Comment:
Otra foma de resolverlo es directamente con `count`

```suggestion
    fun esRobusto() = this.vendedores.count { it.esFirme() } >= 3
```

Reply:
Muy buena esa alternativa, la commiteo asi ya queda en el repo, que la fuerza te acompañe Pablo